src/fastpm.c

```diff
@@ -655,7 +655,8 @@ prepare_cdm(FastPMSolver * fastpm, RunData * prr, MPI_Comm comm)
 static void 
 prepare_ncdm(FastPMSolver * fastpm, RunData * prr, MPI_Comm comm) 
 {
-    if(CONF(prr->lua, n_m_ncdm) == 0) return;
+    if(CONF(prr->lua, n_m_ncdm) == 0
+        || CONF(prr->lua, n_shell) == 0) return; // n_shell condition allows possibility of bkg-only nus
```

Comment:
Let's commit this as a separate PR?

Reply:
Can I just use:

git checkout master
git checkout --patch abacus src/fastpm.c ?

Reply:
Can I just use:

git checkout master
git checkout --patch abacus src/fastpm.c ?

Reply:
done